requirements/django-dev.pip

```diff
@@ -1,2 +1,2 @@
 -r common.pip
-http://github.com/django/django/tarball/master
+https://github.com/django/django/archive/1.11a1.zip
```

Comment:
Is this intentional? Because this will not continue to be the django development version.